lib/location/method_gnss.c

```diff
@@ -721,6 +721,25 @@ static uint8_t method_gnss_tracked_satellites(const struct nrf_modem_gnss_pvt_da
 	return tracked;
 }
 
+static uint8_t method_gnss_tracked_satellites_nonzero_cn0(
```

Comment:
What about combining this with the `method_gnss_tracked_satellites()` function? There's two almost identical functions/loops. You could have only one function and give two variables as pointers, to which the satellite counts are returned to.

Reply:
I didn't notice that `method_gnss_tracked_satellites()` is called from two places. It's still possible to combine the functions, but I leave it up to you if you want to do it or not.